sdk/provisioning/Azure.Provisioning.CloudMachine/tests/CloudMachineTests.cs

```diff
@@ -14,12 +21,382 @@ public class CloudMachineTests
     [Test]
     public void GenerateBicep()
     {
+        CloudMachineInfrastructure? infra = default;
         CloudMachineCommands.Execute(["-bicep"], (CloudMachineInfrastructure infrastructure) =>
         {
+            infra = infrastructure;
             infrastructure.AddFeature(new KeyVaultFeature());
             infrastructure.AddFeature(new OpenAIModel("gpt-35-turbo", "0125"));
             infrastructure.AddFeature(new OpenAIModel("text-embedding-ada-002", "2", AIModelKind.Embedding));
-        }, exitProcessIfHandled:false);
+        }, exitProcessIfHandled: false);
+
+        string bicep = infra!.Build().Compile().FirstOrDefault().Value;
+
+        Assert.AreEqual($$"""
```

Comment:
Can we put the baseline biceps into files as opposed to having them here inline?

Reply:
good idea